test/test_institutions.rb

```diff
@@ -18,43 +21,47 @@ def test_get_with_options
 
   def test_get_invalid_parameters
     assert_raises(Plaid::InvalidRequestError) do
-      client.institutions.get(count: BAD_STRING, offset: BAD_STRING)
+      client.institutions.get(count: BAD_STRING,
+                              offset: BAD_STRING,
+                              country_codes: ['US'])
```

Comment:
I'm wondering if we should have a test for what happens if we use an empty country_codes array